django_ical/tests/test_feed.py

```diff
@@ -41,6 +41,28 @@ def items(self):
             },
             'geolocation': (37.386013, -122.082932),
             'organizer': 'john.doe@example.com',
+            'participants': [
+                {
+                    'name': 'Joe Unresponsive',
+                    'email': 'joe.unresponsive@example.com',
+                    'participation_status': 'NEEDS-ACTION'
+                },
+                # {
```

Comment:
Why are these commented out at the moment?

Reply:
Just trying to get the first participant to pass, others should fall in line.